### PR TITLE
Merge dev-hi into main : Hindi localization going live

### PR DESCRIPTION
**Pull request by Garima-Negi**
- requirements to go live have been completed : [Tracking issue](https://github.com/cncf/glossary/issues/270)

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Thank you all for your devotion and hard work on this project!

@Garima-Negi GitHub says that there exist conflicts.
Please resolve the conflicts!

My suggestion is: Keep `CODEOWNERS` file [as-is in `main` branch](https://github.com/cncf/glossary/blob/main/CODEOWNERS).
```CODEOWNERS
# This CODEOWNERS file defines contributors that are
# responsible for code in this repository.

# Code owners are automatically requested for review
# when someone opens a pull request that modifies code that they own.

# These owners will be default owners for everything in this repository.
# These owners consist of Maintainers and English approvers.
* @caniszczyk @CathPag @jasonmorgan @seokho-son @iamNoah1 @jihoon-seo


# These are the owners (approvers) for localization contents
# in each `/content/language/` directory.

# Approvers for Korean contents
/content/ko/ @seokho-son @Eviekim @jihoon-seo @yunkon-kim

# Approvers for Portuguese contents
/content/pt-br/ @edsoncelio @brunoguidone @jessicalins

# Approvers for Hindi contents
/content/hi/ @Garima-Negi @sayantani11 @anubha-v-ardhan @jayesh-srivastava

# Approvers for German contents
/content/de/ @iamNoah1 @DaveVentura @Petria3s

# Approvers for Italian contents
/content/it/ @fsbaraglia @meryem-ldn @annalisag-spark @sistella

# Approvers for Arabic contents
/content/ar/ @TarekMSayed @same7ammar @AShabana @hacktron95

# Approvers for Bengali contents
/content/bn/ @mitul3737 @Mouly22 @ikramulkayes

# Approvers for Spanish contents
/content/es/ @CathPag @raelga @electrocucaracha

# Approvers for Chinese contents
/content/zh/ @hanyuancheung @Jacob953 @Rocksnake @Submarinee
```

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Comment by seokho-son** [comment]
Check list for maintainers for a PR to make localization live.
 - [ ] validate all files which are not in the localization directory (content/xx/*)
 - [ ] check i18n/xx.toml exist
 - [ ] check l10n configuration in config.toml
 - [ ] check Preview works

File: content/hi/_index.md
Comment (seokho-son) [inline content/hi/_index.md:19]:
```suggestion

Localization of the Cloud Native Glossary was initiated by [Hindi localization team](https://cloud-native.slack.com/archives/C02PCHEQXK6) and contributed by contributions from [contributor1](https://www.linkedin.com/in/... or another link), ..., [contributorN](https://www.linkedin.com/in/seokho-son/), and many more contributors.
```

@Garima-Negi @anubha-v-ardhan 

Please add acknowledgement for major contributors for the Hindi localization :)

I think this update can be done in another PR.

Comment (Garima-Negi) [reply content/hi/_index.md:19]: PR for ack of hindi localization team : [https://github.com/cncf/glossary/pull/664/](https://github.com/cncf/glossary/pull/664/)

Comment (Garima-Negi) [reply content/hi/_index.md:19]: merged!

**Comment by seokho-son** [comment]
@Garima-Negi 
do you want to include `[hi] L10n of Feedback section` #665 ?

anyway, CODEOWNERS has been changed in main branch :) Will incur a conflict

**Comment by Garima-Negi** [comment]
> @Garima-Negi do you want to include `[hi] L10n of Feedback section` #665 ?
> 
> anyway, CODEOWNERS has been changed in main branch :) Will incur a conflict

- is #665 ready to merge? if its a blocker we can wait for its merge to dev-hi.

- for codeowners, do I have push a commit again? The more we wait, the more it gets edited :-)   and pushes us back

**Review by seokho-son — APPROVED** [review]
LGGGGGGGGTM :)

**Pull request merged**
